src/main/java/com/kryptnostic/conductor/rpc/odata/Table.java

```diff
@@ -137,13 +138,44 @@ static CassandraTableBuilder createTableDefinition( Table table ) {
                         .ifNotExists()
                         .partitionKey( NAME )
                         .columns( SECURABLE_OBJECTID );
+
+            // TODO: ENTITY_ID( DataType.uuid() ) or ENTITYID( DataType.text() )
+            case AUDIT_LOG:
+                return new CassandraTableBuilder( AUDIT_LOG )
+                        .ifNotExists()
+                        .partitionKey(
+                                CommonColumns.ACL_KEYS,
+                                CommonColumns.EVENT_TYPE
+                        )
+                        .clusteringColumns(
+                                CommonColumns.PRINCIPAL_TYPE,
+                                CommonColumns.PRINCIPAL_ID,
+                                CommonColumns.TIME_UUID
+                        )
+                        .columns(
+                                CommonColumns.ENTITY_ID,
```

Comment:
I prefer the first one, I may even just use `DATA_ID` + `AUDIT_ID`, although I think all choices here are pretty reasonable